app/utils/phase1_multimodal_extraction_util.py

```
@@ -0,0 +1,1868 @@
+                        f"{rel_path}|sheet={ws.title}|table|rows={chunk_start+1}-{chunk_start+len(section)}"
+                    )
+                    chunks.append({"id": cid, "content": text, "metadata": meta})
+                    stats["table_chunks"] += 1
+
+            # Embedded images in Excel sheets
+            images = list(getattr(ws, "_images", []))
+            for img_i, img in enumerate(images, 1):
+                try:
+                    image_bytes = img._data()
+                except Exception:
+                    continue
```

Ruff (0.15.2)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)